tests/vcs/git/CISkipInspectBearTest.py

```diff
@@ -0,0 +1,139 @@
+        :param kwargs: Keyword arguments to forward to the run function.
+        :return:       A list of the message strings.
+        """
+        dep_bear = GitCommitMetadataBear(None, self.section, Queue())
+        deps_results = dict(GitCommitMetadataBear=list(dep_bear.run()))
+        return list(result.message for result in self.uut.run(
+            deps_results, *args, **kwargs))
+
+    def setUp(self):
+        self.msg_queue = Queue()
+        self.section = Section('')
+        self.uut = CISkipInspectBear(None, self.section, self.msg_queue)
+
+        self._old_cwd = os.getcwd()
+        self.gitdir = mkdtemp()
+        os.chdir(self.gitdir)
+        GitCommitBearTest.run_git_command('init')
+        GitCommitBearTest.run_git_command(
+            'config', 'user.email coala@coala.io')
+        GitCommitBearTest.run_git_command('config', 'user.name coala')
+
+    @staticmethod
+    def _windows_rmtree_remove_readonly(func, path, excinfo):
+        os.chmod(path, stat.S_IWRITE)
+        func(path)
+
+    def tearDown(self):
+        os.chdir(self._old_cwd)
+        if platform.system() == 'Windows':
+            onerror = self._windows_rmtree_remove_readonly
+        else:
+            onerror = None
+        shutil.rmtree(self.gitdir, onerror=onerror)
+
+    def test_skipci_build_commit(self):
+        Path('file1.txt').touch()
+        run_shell_command('git add file1.txt')
+        run_shell_command('git commit -m "Add file1"')
+        self.assertEqual(self.run_uut(
+            ci_engines_used=['AppveyorCI', 'CircleCI', 'TravisCI'],
+            filename_patterns=['Test.py', 'coafile']), [])
+
+        Path('file2.txt').touch()
+        run_shell_command('git add file2.txt')
+        run_shell_command('git commit -m "Add file2"')
+        exception_msg = r'GitLabCI is not supported by CISkipInspectBear.'
+        with self.assertRaisesRegex(RuntimeError, exception_msg):
+            self.run_uut(ci_engines_used=['GitLabCI', 'TravisCI'],
+                         filename_patterns=['Test.py', 'coafile'])
+
+        Path('file4.txt').touch()
+        run_shell_command('git add file4.txt')
+        run_shell_command('git commit -m "Add file4  [ci skip]"')
+        self.assertEqual(self.run_uut(
+            ci_engines_used=['CircleCI', 'TravisCI', 'AppveyorCI'],
+            filename_patterns=['Test.py', 'coafile']), [])
+
+        Path('file5.txt').touch()
+        run_shell_command('git add file5.txt')
+        run_shell_command('git commit -m "Add file5  [ci skip]"')
+        self.assertEqual(self.run_uut(
+            ci_engines_used=['CircleCI', 'TravisCI', 'AppveyorCI'],
+            filename_patterns=['Test.py', 'coafile']), [])
+
+        Path('AbcBearTest.py').touch()
+        run_shell_command('git add AbcBearTest.py')
+        run_shell_command(
+            'git commit -m "AbcBearTest.py: Add first commit\n\n[ci skip]"')
+        self.assertEqual(self.run_uut(
```

Comment:
move `self.run_uut(` to the next line, indented, then you wont have horrible identation below